src/components/Chat/EmojiPickerPopover.tsx

```diff
@@ -5,31 +5,37 @@ import { ClowdrState } from "../../ClowdrTypes";
 
 interface EmojiPickerPopoverProps {
     appState: ClowdrState | null;
+    theme: string | null;
 }
 interface EmojiPickerPopoverState {
+    theme: string | null;
 }
 class EmojiPickerPopover extends React.Component<EmojiPickerPopoverProps, EmojiPickerPopoverState>{
     private pickerRef: React.RefObject<any>;
     constructor(props: EmojiPickerPopoverProps) {
         super(props);
-        this.state = {}
+        this.state = {
+            theme: window.localStorage.getItem("theme") === "0" ? "light" : window.localStorage.getItem("theme")
```

Comment:
This line of code appears to be broken. Did you mean the following:

`theme: !window.localStorage.getItem("theme") ? "light" : window.localStorage.getItem("theme")`

?

Reply:
Alternatively, this would be the more modern way to implement it:

`theme: window.localStorage.getItem("theme") ?? "light"`